scripts/setup-dev.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-ln -s /opt/kazoo/utils/sup/sup /usr/bin/sup
+ln -s /opt/kazoo/core/priv/sup /usr/bin/sup
```

Comment:
This symlink will fail on systems that already have /usr/bin/sup set (so it will not work to migrate from an existing install).

This is the same for all of the symlinks set in this file, I think that should probably be changed so that it always re-creates the symlinks.